src/cf-layout/src/molecules/heroes.less

```diff
@@ -0,0 +1,128 @@
+@hero-desktop-height: 285px;
+@hero-image-height: 195px;
+@hero-bg: #f7f8f9;
```

Comment:
Add corresponding 18F standards variables in a comment to match the rest of the project.